src/CodeGen_ARM.cpp

```diff
@@ -2468,6 +2503,30 @@ string CodeGen_ARM::mattrs() const {
     if (target.has_feature(Target::ARMv81a)) {
         attrs.emplace_back("+v8.1a");
     }
+    if (target.has_feature(Target::ARMv82a)) {
+        attrs.emplace_back("+v8.2a");
+    }
+    if (target.has_feature(Target::ARMv83a)) {
+        attrs.emplace_back("+v8.3a");
+    }
+    if (target.has_feature(Target::ARMv84a)) {
+        attrs.emplace_back("+v8.4a");
+    }
+    if (target.has_feature(Target::ARMv85a)) {
+        attrs.emplace_back("+v8.5a");
+    }
+    if (target.has_feature(Target::ARMv86a)) {
+        attrs.emplace_back("+v8.6a");
+    }
+    if (target.has_feature(Target::ARMv87a)) {
+        attrs.emplace_back("+v8.7a");
+    }
+    if (target.has_feature(Target::ARMv88a)) {
+        attrs.emplace_back("+v8.8a");
+    }
+    if (target.has_feature(Target::ARMv89a)) {
+        attrs.emplace_back("+v8.9a");
+    }
```

Comment:
If `target` has been completed at this point, isn't this going to add attrs for everything requested and below? Is LLVM cool with that?

Reply:
Uh, good question.